ext/Hash-Util/lib/Hash/Util.pm

```diff
@@ -39,10 +39,11 @@ our @EXPORT_OK  = qw(
                      used_buckets
                      num_buckets
                     );
+our $VERSION;
```

Comment:
Any reason to hoist the `our` outside like this?

Reply:
This is a slight blead over from the strict/warnings work. I think we opted not to fix it.

Reply:
This is an incorrect definition of a global variable... 
```
use strict;
use warnings;

BEGIN {
	our $x = q[blabla];
}

print qq[$x\n];
```
once run it raise a warning
```
╰─> perl test.pl
Variable "$x" is not imported at test.pl line 8.
Global symbol "$x" requires explicit package name (did you forget to declare "my $x"?) at test.pl line 8.
Execution of test.pl aborted due to compilation errors.
```